codice/ddf
# Ddf 2481 add windows commands to installation instructions

#### What does this PR do?

Copy edits to quickstart and installation instructions.

#### Who is reviewing it?

@mcalcote @clockard @tbatie @garrettfreibott @emanns95 @mweser @spearskw

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@pklinef
@shaundmorris

#### How should this be tested?

Verify changes to content.

[DDF-2481](https://codice.atlassian.net/browse/DDF-2481)

#### Checklist:
